projects/components/src/confirmation/confirmation-modal.component.scss

```diff
@@ -15,6 +15,8 @@
     @include body-1-regular($gray-7);
     text-align: center;
     word-break: break-word;
+    flex: 1 1 auto;
+    overflow: hidden;
```

Comment:
What is the reason behind this?

Reply:
This guy doesn't expand and takes up all the available space. So, if you put any custom content via the template as a child of this element, then its container height would be limited and the content would overflow. In order to show a scrollbar in its child template, we need to add this change here.